modules/java/src/main/java/org/syslog_ng/LogDestination.java

```diff
@@ -26,12 +26,11 @@
 
 public abstract class LogDestination extends LogPipe {
 
-	protected static final int WORKER_INSERT_RESULT_DROP = 0;
-	protected static final int WORKER_INSERT_RESULT_ERROR = 1;
-	protected static final int WORKER_INSERT_RESULT_EXPLICIT_ACK_MGMT = 2;
-	protected static final int WORKER_INSERT_RESULT_SUCCESS = 3;
-	protected static final int WORKER_INSERT_RESULT_QUEUED = 4;
-	protected static final int WORKER_INSERT_RESULT_NOT_CONNECTED = 5;
+	protected static final int DROP = 0;
+	protected static final int ERROR = 1;
+	protected static final int SUCCESS = 3;
+	protected static final int QUEUED = 4;
+	protected static final int NOT_CONNECTED = 5;
 
 	public LogDestination(long pipeHandle) {
```

Comment:
I like these names a lot better. I would love to see the c names to be simplified as well. But that's optional.